### DISCUSSION TITLE
Creadores de contenido para ayudar a nuevos integrantes

### DISCUSSION THREAD

**Discussion by Ignacioskm** [Launch-X-Latam/MisionFrontEnd#84] [General]
Buenas compañeros 👋
Esperando que se encuentren muy bien se me ocurrió una idea para ayudarnos entre todos, ya que en este Bootcamp hay gente nueva como gente mas experimentada en la programación, me gustaría dejar este espacio para que compartan sus creadores de contenido favoritos en el ámbito de la programación. 💻

Partiré yo:

"midudev" tanto en <a href="https://www.youtube.com/c/midudev/videos">YouTube</a> como en <a href="https://www.tiktok.com/@midudev?lang=es">TikTok</a> , sube bastante contenido, tutoriales, trucos para optimización de código y consejos laborales.  🚀

**Comment by SilvyeCG** [comment]
La Wikipedia de Ernestesto creo que también ofrece buenos cursos para niveles iniciales y ofece cursos desde desarrollo de lógica, algoritmos, hasta lenguajes más especificos como java, python, js, etc.

**Reply by Ignacioskm** [reply]
Gracias!

**Comment by Ricardo-g-garcia** [comment]
Canal de Youtube esta en ingles pero tiene muy buen contenido
https://www.youtube.com/c/Freecodecamp

y en español freecodecamp en español 

https://www.youtube.com/c/freeCodeCampEspa%C3%B1ol/featured

**Reply by Ignacioskm** [reply]
Buenisimo freecodecamp para partir y super práctico!

**Comment by maxkario** [comment]
Yo aportaré estos dos sitios. 

https://lenguajehtml.com/

https://aprendejavascript.org/

**Comment by Mar-rbw** [comment]
Yo aportare con estos videos practicos de Escalab Academy
https://www.youtube.com/watch?v=b2vB7SoMmU0&list=PLfrAWbOpzg7zMV2eiSkGnQc-9cvCsN2Px

**Comment by LaMazorca** [comment]
Pues yo puedo aportar que LinkedIn tiene un curso completo de fullstack, gratuito y con certificación, en LinkedIn learning, por si alguien le interesa

**Reply by LaMazorca** [reply]
[https://www.linkedin.com/learning/paths/conviertete-en-desarrollador-web-full-stack](https://www.linkedin.com/learning/paths/conviertete-en-desarrollador-web-full-stack), perdón aquí les dejo el Link, que les manda a ese curso de Linkedln

**Comment by Ireneripo-LaunchX** [comment]
Gracias a todos por sus aportes!! A mí me gusta la manera de explicar de Código con Juan, tiene algunos cursos gratis y otros de pago, pero accediendo a través de su página hay cupones de descuento y cualquiera de sus cursos queda en 10USD.

https://codigoconjuan.com/

**Reply by Ignacioskm** [reply]
No lo conocía, muchas gracias 😄

**Comment by JorgeDevmm** [comment]
Aquí dejo algunas de mis recomendaciones tanto como en desarrollo frontend y backend espero les sirva:

gratuito sin duda de los mejores te dejan excelentes bases para poder practicar por nuestra cuenta:

https://www.youtube.com/c/jonmircha

https://www.youtube.com/c/pildorasinformaticas

https://www.youtube.com/c/freeCodeCampEspa%C3%B1ol

https://www.youtube.com/c/Bluuweb

https://www.freecodecamp.org/


de pago aunque la inversión es mínima 10 dolares cada curso de estos instructores en español e ingles lo valen:

https://fernando-herrera.com/#/

https://codigoconjuan.com/

https://codingheroes.io/

**Comment by LauraHenao27** [comment]
Para mi https://www.youtube.com/c/FaztTech, explica muy bien
